### ISSUE TITLE
ghcide 8.6.5 tests don't pass

### ISSUE THREAD

**Issue by rvl** [hercules-ci/haskell-language-server.nix#5]
When I attempt to install `ghcide` or `haskell-language-server` with:

    nix-env -f default.nix -iA ghc-8_6_5.ghcide
    nix-env -f default.nix -iA ghc-8_6_5.haskell-language-server

The test suite does not pass. Some of the errors look like they are caused by running the tests in the nix build sandbox (e.g. missing home directory).

However I can ignore the tests and install the executables directly with:

    nix-env -f default.nix -iA ghc-8_6_5.ghcide.exes.ghcide
    nix-env -f default.nix -iA ghc-8_6_5.haskell-language-server.exes.haskell-language-server

**Comment by oneingan** [comment]
how do you workaround the `fatal: not a tree object: fb3859dca2e54d1bbb2c873e68ed225fa179fbef` error ? https://github.com/hercules-ci/haskell-language-server.nix/pull/2

**Comment by rvl** [comment]
I thought we had fixed that. I just built from master and it worked, but maybe the git repo updated but I had got the source from hercules-ci.cachix.org. You are building from the master branch, right?

**Comment by oneingan** [comment]
Yep, I clone the repo and run `nix-env -f default.nix -iA ghc-8_6_5.ghcide` command. Also tried to update all sources with `update` script. Same error.
